### ISSUE TITLE
foreman::install pulling in missing rpm

### ISSUE THREAD

**Issue by jadestorm** [theforeman/puppet-foreman#860]
Hi folk!  I'm attempting to switch from the memcached plugin to redis for the rails cache -- and the puppet module is attempting to pull in, well:

```
if $foreman::rails_cache_store['type'] == 'redis' {
 package { 'foreman-redis':
 ensure => installed,
 }
}
```

However, that module does not exist in the yum repos.  (RHEL7)  I can't find anything in the documentation that tells me anything special I'm supposed to be doing here.  I simply followed the guidance from the README.  Basically this:

```
  rails_cache_store => {
    'type' => 'redis',
    'urls' => ['localhost:8479/0'],
    'options' => {
      'compress' => 'true',
      'namespace' => 'foreman'
    }
  }
```

(all that I have different is the array of urls)

Any idea why the rpm is missing?  Is it -supposed- to be missing?  Is there a workaround I can use
for now?  Or if you all have a quick fix I can certainly update to a git commit instead of a released version.

Thanks!

**Comment by ehelms** [comment]
What Foreman repositories are you using? The foreman-redis package was introduced as part of the 2.1 release which is currently in RC phase.

**Comment by jadestorm** [comment]
Hi @ehelms !   Hrm  I upgraded to the 15.0.1 plugin version since it indicates it is meant for foreman 2.0+.  I am currently running foreman 2.0.1.  It sounds like I should not be doing that quite yet?  I reverted to using memcached btw, so no huge deal,  but there wasn't anything in the module readme that I could see that indicated I shouldn't be using redis support yet.

To answer your question though -- I'm using:
baseurl=https://yum.theforeman.org/releases/2.0/el7/$basearch
and such.

If the answer is literally "wait for foreman 2.1.0", I suppose all this issue is is "would someone mind adding "Starting with Foreman 2.1.0, you can use redis for the rails cache..." something like that.  =)

**Comment by jadestorm** [comment]
(but you are welcome to close it  =)  )

**Comment by ehelms** [comment]
Essentially that is the answer as far as I know. We sometimes miss changes like this as we make updates to master of this module to support newer Foreman versions and balancing this with the lifecycle of the module itself.

**Comment by jadestorm** [comment]
Ok!  No worries.  I'll hang tight until 2.1.0 comes out!  Thanks!  (going to go ahead and close this)

**Comment by ekohl** [comment]
This was actually a bug and it was supposed to be published in 2.0. https://github.com/theforeman/foreman-packaging/commit/d1e2c4dea8b58434da0a25410e561f6d6a1eef21 fixes it by adding foreman-redis to comps.xml. That (or at least part of it) should be cherry picked to 2.0.
